registry/client/src/sharedLibs/List.js

```diff
@@ -1,23 +1,13 @@
 import React from 'react';
 import { useMediaQuery } from '@material-ui/core';
-import {
-    Datagrid,
-    List,
-    SimpleList,
-    TextField,
-    EditButton,
-    FunctionField,
-} from 'react-admin'; // eslint-disable-line import/no-unresolved
-import {
-    Empty,
-    ListBulkActions,
-    ListActionsToolbar,
-} from '../components';
+import { Datagrid, List, SimpleList, TextField, EditButton, FunctionField } from 'react-admin'; // eslint-disable-line import/no-unresolved
```

Comment:
please avoid import/no-unresolved looks scary.

Reply:
Sure, done.
BTW, i don't see any eslint-related checks in the repo 🤔